<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messages</class>
 <widget class="QDialog" name="Messages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupe_reception">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>  Boite de Reception    </string>
   </property>
   <widget class="QLabel" name="label_dest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Destinataire</string>
    </property>
   </widget>
   <widget class="QLabel" name="dest_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_heure">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Heure et Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="heure_text">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="message_reception">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="but_lu">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_quitter">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupe_envoi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>621</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>  Boite d'Envoi    </string>
   </property>
   <widget class="QLabel" name="label_destination">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_dest">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_env">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_source">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_message">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_destination_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Envoyer en tant que</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
